libs/ledger-live-common/src/transactionsAlerts/index.ts

```diff
@@ -0,0 +1,51 @@
+import type { ChainwatchNetwork, Account } from "@ledgerhq/types-live";
+import ChainwatchAccountManager from "./ChainwatchAccountManager";
+
+const formatAccountsByCurrencies = (newAccounts: Account[], removedAccounts: Account[]) => {
```

Comment:
This function is easy to test, and given that we're dealing with LLC it might be a good idea to run 1/2 tests :)

Reply:
Totally ! Will do that this week and push them in another PR